### PR TITLE
chore: update pricing

### PR DESCRIPTION
**Pull request by MildTomato**
#### Database
- guidance on backups > 1TB in size
#### Auth
- added “advanced security features”
- MAUs changed to unlimited for Enterprise
#### Realtime
- Added new realtime section and pricing
#### Edge Functions
- Added extra functions cost for Pro tier
- Runtime hours removed
#### Support
- Security features moved to own section
#### Platform Security and Compliance
- New section made
- added Access roles
- added Custom Domains
- added Vanity URLs added
#### Database Compute Add ons
- New label “Optimized Compute”

### CODE REVIEW

File: apps/www/data/Pricing.json [hunk 1]
```diff
@@ -262,92 +284,124 @@
...
         }
       },
       {
-        "title": "Email support SLA",
+        "title": "SOC2",
         "tiers": {
           "free": false,
           "pro": false,
-          "enterprise": true
+          "enterprise": "Coming soon"
```
Comment (awalias) [inline apps/www/data/Pricing.json:328]: how come we're switching this one?

File: apps/www/data/Pricing.json [hunk 2]
```diff
@@ -262,92 +284,124 @@
...
-          "enterprise": "90 days"
+          "free": "Owner, Developer",
+          "pro": "Owner, Developer",
+          "enterprise": "Additional owner(s), admin, read-only, billing admin, custom"
         }
       },
       {
-        "title": "Log drain",
+        "title": "Vanity URLs",
+        "tooltip": "We currently screen vanity URLs to ensure they are non-offensive. Please email growth@supabase.io with your project details to request your vanity URL. We process requests in batches once every 14 days.",
```
Comment (awalias) [inline apps/www/data/Pricing.json:357]: can probably drop 'We currently screen vanity URLs to ensure they are non-offensive.'

**Review by awalias — APPROVED** [review]

File: apps/www/data/Pricing.json (hunk 2 above)
Comment (MildTomato) [reply apps/www/data/Pricing.json]: done

File: apps/www/data/Pricing.json (hunk 1 above)
Comment (MildTomato) [reply apps/www/data/Pricing.json]: oh that was a mistake

**Pull request merged**